rabbitmq/README.md

```diff
@@ -167,91 +167,14 @@ Additional helpful documentation, links, and articles:
-| rabbitmq.queue.messages\_ready                                 | rabbitmq\_queue\_messages\_ready                                                                                              |                     |
-| rabbitmq.queue.messages\_unacknowledged                        | rabbitmq\_queue\_messages\_unacked                                                                                            |                     |
-
-The following Management plugin metrics to our knowledge have no equivalent in the Prometheus plugin.
-
-- rabbitmq.connections.state
-- rabbitmq.exchange.messages.ack.count
-- rabbitmq.exchange.messages.ack.rate
-- rabbitmq.exchange.messages.confirm.count
-- rabbitmq.exchange.messages.confirm.rate
-- rabbitmq.exchange.messages.deliver\_get.count
-- rabbitmq.exchange.messages.deliver\_get.rate
-- rabbitmq.exchange.messages.publish.count
-- rabbitmq.exchange.messages.publish.rate
-- rabbitmq.exchange.messages.publish\_in.count
-- rabbitmq.exchange.messages.publish\_in.rate
-- rabbitmq.exchange.messages.publish\_out.count
-- rabbitmq.exchange.messages.publish\_out.rate
-- rabbitmq.exchange.messages.redeliver.count
-- rabbitmq.exchange.messages.redeliver.rate
-- rabbitmq.exchange.messages.return\_unroutable.count
-- rabbitmq.exchange.messages.return\_unroutable.rate
-- rabbitmq.node.partitions
-- rabbitmq.node.run\_queue
-- rabbitmq.node.running
-- rabbitmq.overview.messages.ack.count
-- rabbitmq.overview.messages.ack.rate
-- rabbitmq.overview.messages.confirm.rate
-- rabbitmq.overview.messages.deliver\_get.rate
-- rabbitmq.overview.messages.publish.rate
-- rabbitmq.overview.messages.publish\_in.count
-- rabbitmq.overview.messages.publish\_in.rate
-- rabbitmq.overview.messages.publish\_out.count
-- rabbitmq.overview.messages.publish\_out.rate
-- rabbitmq.overview.messages.redeliver.rate
-- rabbitmq.overview.messages.return\_unroutable.rate
-- rabbitmq.overview.queue\_totals.messages.rate
-- rabbitmq.overview.queue\_totals.messages\_ready.rate
-- rabbitmq.overview.queue\_totals.messages\_unacknowledged.rate
-- rabbitmq.queue.active\_consumers
-- rabbitmq.queue.bindings.count
-- rabbitmq.queue.messages.ack.count
-- rabbitmq.queue.messages.ack.rate
-- rabbitmq.queue.messages.deliver.count
-- rabbitmq.queue.messages.deliver.rate
-- rabbitmq.queue.messages.deliver\_get.count
-- rabbitmq.queue.messages.deliver\_get.rate
-- rabbitmq.queue.messages.publish.rate
-- rabbitmq.queue.messages.rate
-- rabbitmq.queue.messages.redeliver.rate
-- rabbitmq.queue.messages\_ready.rate
-- rabbitmq.queue.messages\_unacknowledged.rate
+### Migrating to Prometheus Plugin
+
+When migrating from the *Management Plugin* to the *Prometheus Plugin* check the metrics that you rely on most. Some may have new names, some may not be available anymore.
```

Comment:
```suggestion
When migrating from the Management Plugin to the Prometheus Plugin, check the metrics that you rely on most because some of them might have new names or might not be available anymore.
```